deploy/release.yaml

```diff
@@ -95,7 +95,7 @@ spec:
       serviceAccountName: knative-openshift-ingress
       containers:
         - name: knative-openshift-ingress
-          image: quay.io/openshift-knative/knative-openshift-ingress:v0.0.5
+          image: quay.io/openshift-knative/knative-openshift-ingress:v0.0.6
```

Comment:
Is there a 0.0.6 release with these changes now? If so, there are a couple of other places to bump as done in things like https://github.com/openshift-knative/knative-openshift-ingress/commit/0251d3eb6e81e013459093499cacf226d2569b28

Reply:
Changes pushed.